src/pages/resume.js

```diff
@@ -45,270 +48,145 @@ const Resume = () => (
-            highlight I'm currently working on is Vanilla Framework, a SASS
-            based framework of patterns to support all of Canonical&#039;s
-            upcoming web releases.
+          In my role as an Application Developer with Citi, 
+          I work with a development team to deliver continuous stability and enhancement to the e – Trading platform. 
+          I have been a liaison with global business and development teams, 
+          coordinating the testing of new and upgraded market-data components.
           </p>
         </div>
       </section>
       <section className="entry-list__item row">
         <div className="column">
           <span className="entry-list__meta">
             <time datetime="2015-10-16">Oct</time> &ndash;
-            <time datetime="2015-12-18">Dec 2015 (Short-term contract)</time>
+            <time datetime="2015-12-18">Jan 2018 (Intern contract)</time>
           </span>
         </div>
         <div className="column column-75">
           <h3 className="entry-list__heading">
-            <img src={madebymanyLogo} alt="Made by Many logo" />
+            <img src={consalisLogo} alt="Consalis logo" />
           </h3>
           <p>
-            <em>Made by Many</em> is an award winning innovation accelerator
-            based in Angel. I'm worked with them over 5 two week sprints to
-            develop a complex Ruby on Rails product with an accompanying Pattern
-            Library for a leading scientific laboratory.
+            <em>Consalis</em> is an Edinburgh based company which supports the
+            investment management industry in Scotland. They specialise in building
+            bespoke software solutions to enhance key business processes.
           </p>
           <p>
-            The lab was a large corporate with multiple competing sub-brands so
-            the challenge was to develop an over-arching set of patterns &amp;
-            living components that could be thread through multiple products to
-            build a cohesive overall brand.
+          I worked as part of a development team to build a web-based asset management application. 
+          This is a front office application for boutique asset managers. 
+          The application compromises of compliance modelling, order generation and reconciliation modules.
+          The user interface was developed with Angular 4, for which I have received training. 
+          The styling of the application makes use of the Bootstrap, enabling a consistent and reactive design.
           </p>
         </div>
       </section>
       <section className="entry-list__item row">
         <div className="column">
           <span className="entry-list__meta">
-            <time datetime="2015-02-04">Feb</time> &ndash;
-            <time datetime="2015-10-16">Oct 2015 (Medium-term contract)</time>
+            <time datetime="2015-02-04">June</time> &ndash;
```

Comment:
You need to make sure the `datetime` attributes match the text values you place inside the tags.